diff --git a/src/bindings/python/wheel/setup.py b/src/bindings/python/wheel/setup.py
index fcaa5e5f..b8eaccc4 100644
--- a/src/bindings/python/wheel/setup.py
+++ b/src/bindings/python/wheel/setup.py
@@ -72,6 +72,12 @@ LIB_INSTALL_CFG = {
         "install_dir": OV_RUNTIME_LIBS_DIR,
         "rpath": LIBS_RPATH,
     },
+    "vpu_plugin": {
+        "name": "myriad",
+        "prefix": "libs.core",
+        "install_dir": OV_RUNTIME_LIBS_DIR,
+        "rpath": LIBS_RPATH,
+    },
     "multi_plugin": {
         "name": "multi",
         "prefix": "libs.core",
